### ISSUE TITLE
Font audit script

### ISSUE THREAD

**Issue by g-a-v-i-n** [urbit/PaperRenderer#103]
We have an issue where it is hard to see what fonts are included in the templates.json from Figma. It would be helpful to have a script that runs through all the element objects in the templates and returns a list of the the different font-families and weights used. Keep in mind that fonts are their font-weights + font-families.

**Comment by francescanarea** [comment]
could this be used for preloading? or are we still anti-preloading?  we can preload with fontfaceset

**Comment by francescanarea** [comment]
the only reason why i suggest preloading is because we don't really care if it renders fast since the user isn't ultimately viewing this on web, right?  preloading would allow us to not worry about any font stuff during the drawing process (so theres potentially better flow).  otherwise, the font detection has to happen in the middle of each drawing function (we would have to add in identical lines for font detection/loading in each draw function). 

im picturing this:
1. during convert, each page is given a new attribute called fonts = [array of font-families/weights]
2. before drawing the page, load the fonts (w/ error detection)
3. draw

i also suggest this because if we loaded the font in each draw text method, then we'd be asking to load the same font potentially multiple times (although not sure if the load would be executed multiple times or skipped over after the first time)

tldr: if we want to render PC on a webpage vs. PDF eventually, we shouldn't preload but if thats not in the realm of possibilities, preload could maybe make this lib cleaner

**Comment by francescanarea** [comment]
@g-a-v-i-n let me know your thoughts or if you see some flaws in my logic here!
